StarCraft2.ahk

```diff
@@ -130,7 +137,7 @@ class SC2
         this.tankxPos := xPos
         this.tankyPos := yPos
         this.casterxPos := xPos + 188    ; compared to the center: a little bit to the right
-        this.casteryPos := yPos - 120    ; compared to the center: a little bit upper
+        this.casteryPos := yPos - 25    ; compared to the center: a little bit upper
```

Comment:
That doesn't need to change. You want to fire his "stun rocket thingy" behind you a little, so that zombies will come closer to you.
You want that (zombies coming closer to you) because the closer they are to you, the more of them will get hit by goon q, or my medic grenade (think that mouse pos {X, Y} represents the center of the aoe -- you want as many zombies hit by that aoe spell as possible).

Reply:
I think it's better to add a new pair of coordinates specifically for these defensive spells. (something i will do after merging this PR)